src/platforms/apple/common/enriching-events/user-feedback.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "User Feedback"
+sidebar_order: 105
+
+description: "Learn more about the user feedback API, which gives you the possibility to collect user information when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+---
+
+The user feedback API gives you the possibility to collect user information when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues.
+Sentry needs the `eventId` to be able to associate the user feedback to the corresponding event. To get the `eventId` you for example [`beforeSend`](/platforms/apple/guides/ios/configuration/options/#before-send) or use the return value of the methods capturing an event.
+
+```swift
+import Sentry
+
+let userFeedback = UserFeedack(eventId: /* Put in eventId */)
+userFeedback.comments = "It broke."
+userFeedback.email = "john.doe@example.com"
+userFeedback.name = "John Doe"
+SentrySDK.capture(userFeedback: userFeedback)
+```
+
+```objc
```

Comment:
```suggestion
```objc {tabTitle:Objective-C}
```

Reply:
Nice, I didn't know you can do that. I need to fix it everywhere.

Reply:
yep, you can check the Java docs https://docs.sentry.io/platforms/java/#install
install script for Gradle/Maven and SBT as tabs, it looks cool